Formula/cpufetch.rb

```diff
@@ -0,0 +1,18 @@
+class Cpufetch < Formula
+  desc "Simple yet fancy CPU architecture fetching tool"
+  homepage "https://github.com/Dr-Noob/cpufetch"
+  url "https://github.com/Dr-Noob/cpufetch/archive/v1.00.tar.gz"
+  sha256 "2254c2578435cc35c4d325b25fdff4c4b681de92cbce9a7a36e58ad58a3d9173"
+  license "MIT"
+  head "https://github.com/Dr-Noob/cpufetch.git", branch: "master"
+
+  def install
+    system "make"
+    bin.install "cpufetch"
+    man1.install "cpufetch.1"
+  end
+
+  test do
+    system "cpufetch", "--version"
```

Comment:
We need a functional test as version checks or help checks (`foo --version` or `foo --help`) are not sufficient, as explained in the Formula Cookbook – they often fail to detect broken installs. Could you compare some output that doesn't depend on the system/processor type?

Reply:
I don't think there's any way for me to quickly test this formula in any capacity that doesn't depend on the system. The most comprehensive test I can think of is to use system tools to fetch the same info `cpufetch` does, and compare the two, but this may not produce stable results either.

Reply:
We can compare output that would not change no matter what the system is, like:
```
cpufetch is computing APIC IDs, please wait...
```
and the various headers of the information that is output such as "Name" or "Cores".

Even just running `cpufetch` and ensuring it exits without any error would be fine, like what the `neofetch` formula does.

Reply:
> Even just running `cpufetch` and ensuring it exits without any error would be fine, like what the `neofetch` formula does.

I was thinking about doing just this. After reinstalling, I noticed that there is a `-d` flag for debugging purposes. I could invoke `cpufetch -d` for the test. It simply prints CPU model and CPUID levels, and nothing else.